run_workflow_and_argo.sh

```diff
@@ -9,7 +9,7 @@ else
     echo "$MACHINE_CONFIG" > /tmp/config.yml
 fi
 
-gordo-components workflow generate --machine-config /tmp/config.yml --output-file /tmp/generated-config.yml
+gordo-components workflow generate --machine-config /tmp/config.yml --output-file /tmp/generated-config.yml --log-level INFO
```

Comment:
Nja, I dont think this can now be overridden by an environment variable anymore

Reply:
I removed this, did not make sense after thinking about it.